### ISSUE TITLE
Expand setup configuration

### ISSUE THREAD

**Issue by johnsills1** [huttotw/homebridge-kia-connect#8]
Would it be possible to expand the configuration page to allow us to disable certain aspects of the plugin. For example, the Temperature Sensor must not be available on my Kia Stinger. I have the Kia Connect Ultimate and the only temperature reading I ever see is 32 degrees and it never changes. 

Great work on the Plugin by the way. This is really cool.

**Comment by johnsills1** [comment]
Just to elaborate, homebridge is receiving a NaN (number) from my Stinger for the Temperature reading.

**Comment by asabatino88** [comment]
Agreed - would also be good to extend the remote start time to 10 minutes from 5 minutes. It looks like there's a setting in client.ts that would allow this extension. I also note that the Home Assistant plugin lets you choose 10 minutes. I like the ease of this plugin vs the Home Assistant one, so I haven't fully moved over yet.

**Comment by johnsills1** [comment]
I had another idea for the plugin. Would it be possible to take the battery status percentage and create something like a light sensor so we could use the percentage in an automation to warn us when the battery drops below a certain percentage. I already had to change my car battery after one year (strange) so I’ve been watching this data. It’s just buried in the engine accessory settings

**Comment by huttotw** [comment]
Hey @johnsills1 and @asabatino88 sorry for the delay. I think these would be great additions to the plugin. I think all we would need to do is add some fields to the `config.schema.json` file so they can be filled in by the user upon installation, then we should use those as parameters throughout.

We should take the opportunity to add some tests here. I might work on this shortly!

**Comment by huttotw** [comment]
@johnsills1 let me know what you think of #11

**Comment by johnsills1** [comment]
Looks good to me. I’m not a code writer, but I love beta testing, haha

John Sills

**Comment by asabatino88** [comment]
For what it's worth, I'm also experiencing the NaN on the external temperature sensor (which means it just defaults to 32 degrees all the time). I tried to comment out all of the external temperature sensor code in my forked code base, but wasn't able to stop it from showing up as an accessory in Homebridge. I'm not a dev but have done some basic coding over the years, so I was likely just missing something when playing around with it.

**Comment by johnsills1** [comment]
How are the updates coming? I am hoping we can expose the battery as a light brightness sensor so we can utilize it in automations without jumping through hoops with the Eve, Controller for HomeKit and Home apps. Thanks

**Comment by johnsills1** [comment]
Have you had time to work on this?
